include/libdrizzle-redux/statement.h

```diff
@@ -392,8 +405,22 @@ const char *drizzle_stmt_get_string(drizzle_stmt_st *stmt, uint16_t column_numbe
  * @return The int value
  */
 DRIZZLE_API
-uint32_t drizzle_stmt_get_int(drizzle_stmt_st *stmt, uint16_t column_number,
-                              drizzle_return_t *ret_ptr);
+int32_t drizzle_stmt_get_int(drizzle_stmt_st *stmt, uint16_t column_number,
```

Comment:
Hmm, so this is actually changing the API. In this case it can't go in a minor.

Reply:
Yes. Realized this as well. Though the current API is broken since you can't `set` an signed numeric value for a query parameter. Will create a  `v7.0.0` Milestone and `v7.x.x` branch